homeworks/becseipeter/5_oop/car_fleet_mgr.py

```diff
@@ -0,0 +1,70 @@
+class Car:
+    def __init__(self, brand: str, model: str, year: int):
+        self.brand = brand
+        self.model = model
+        self.year = year
+        self.mileage = 0                    
+        self.fuel_level = 100               
+
+    def drive(self, kilometers):
+        fuel_needed = kilometers * .1
+        if self.fuel_level > fuel_needed:
+            self.fuel_level -= fuel_needed
+            self.mileage += kilometers
+        else:
+            print(f'Find a gas station.')
+
+    def refuel(self, amount):
+        if self.fuel_level + amount > 100:
+            self.fuel_level = 100
+        else:
+            self.fuel_level += amount
+            print(f"{self.brand} {self.model} refueled. Fuel level is now {self.fuel_level}%.")
+
+
+class Fleet:
+    def __init__(self, name):
+        self.name = name
+        self.cars = []  
+
+    def add_cars(self, car):
+        self.cars.append(car)
+
+    def remove_car(self, car):
+        if car in self.cars:
+         self.cars.remove(car)
+
+    def total_mileage(self):
+        return sum(car.mileage for car in self.cars)
+    
+    def __str__(self):
+        return f"Total performance of the fleet: {self.total_mileage()} km"
+
+
+###############################################################################################################################
+  
+    car1 = Car("Audi", "Quattro", 1980)
```

Comment:
Így ettől a sortól egészen a 68. sorig minden a Fleet osztály része, mert beljebb kezded a sorokat.
Ettől a kódod le sem fut addig amíg ezt egy tab-bal kijjebb nem rakjuk, mivel az 51. soron lévő Fleet()-et nem tudja hova tenni az osztályon belül.

![image](https://github.com/user-attachments/assets/76462489-6bdd-4172-a5c4-1e2eaf9b87b3)
mozgass mindent egy tab-bal kijjebb:
![image](https://github.com/user-attachments/assets/72e2b44f-10ca-4641-99cf-704fa2c48f62)

Jelöld ki a 46. sortól a végéig az egészet és SHIFT+TAB.

plusz a Fleet-nek kell egy name paraméter is, ezt is add át.